ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -34,6 +45,9 @@ import AccessControlBreadcrumbs from '../AccessControlBreadcrumbs';
 import AccessControlHeading from '../AccessControlHeading';
 import AccessControlHeaderActionBar from '../AccessControlHeaderActionBar';
 import usePermissions from '../../../hooks/usePermissions';
+import { integrationsPath } from '../../../routePaths';
+import { getVersionedDocs } from '../../../utils/versioning';
+import useMetadata from '../../../hooks/useMetadata';
```

Comment:
Aha, these 3 plus the 1 immediately preceding have pseudo-absolute paths:
* which follow installed packages
* which precede relative imports from within the same Containers folder

Can you move up and interleave in an intuitive order among the `import` statements lines 27 through 34?

routePaths is a judgement call what is intuitive, because it is in src folder.

```ts
import usePermissions from 'hooks/usePermissions';
import { integrationsPath } from 'routePaths';
import { getVersionedDocs | from 'utils/versioning';
import useMetadata from 'hooks/useMetadata';
```